server/helios/models.py

```diff
@@ -1294,3 +1294,9 @@ def verify_decryption_proofs(self):
             self.public_key,
             algs.EG_fiatshamir_challenge_generator,
         )
+
+class QrCode(models.Model):
```

Comment:
@KonradSwierczynski  @piotrszyma 
What are the demands regarding the elections? 
We can point in `QrCode` model to both election and user (owner of the code) - this would allow us to perform many elections.

Reply:
Ok - after the discussion we chose to link it to the `Voter` entity.

Reply:
Could you explain why you've made this decision here?

Reply:
What decision?
The one regarding usage of Voter was made on the basis of the talk with @KonradSwierczynski  and @adambudziak.